cypress/integration/non-regression-tests-suite/homepage.spec.js

```diff
@@ -49,12 +49,12 @@ describe(`Home page tests : Tests execution date and time : ${new Date()}`, () =
 		cy.verifySocialMediaLink(4, Navlinks.youtube);
 	});
 
-	it.skip("Has no detectable a11y violations on load (filtering to only include critical impact violations)", () => {
+	/*it("Has no detectable a11y violations on load (filtering to only include critical impact violations)", () => {
```

Comment:
Worth putting in a comment to say why this is disabled so we have it in the code?

Reply:
Issues have been fixed and test is not throwing accessibility violation error now. Therefore I've removed the skip tag from the tests.